tests/integration/targets/idrac_certificates/tests/_wait_for_idrac_readiness.yml

```
@@ -0,0 +1,14 @@
+---
+- name: Wait for port to become open on the host
+  ansible.builtin.wait_for:
+    host: "{{ idrac_ip }}"
```

I see there is support for ipv6 in module and in some TC files however it missing at some place.
Please check and updates in all the other places.

```suggestion
    host: "{{ idrac_ip|ansible.utils.ipwrap }}"
```

---

done